File: datasets/IR-Plag/intellij-original-kotlin-rest-java/case-05/plagiarized/L2/04/method.java
package IR;

import java.util.*;

class method{
	//prog utama
	public static void main(String[] args) 
	{
			System.out.print("Enter an integer: ");
			Scanner input = new Scanner(System.in);
		int n = input.nextInt();
		//pamggil method
			beautyReverse(n);
	}
	
	//method reverse
	public static void beautyReverse(int num) 
	{
		while (num != 0)
			{
			int r = num % 10;
				System.out.print(r);
			num = num / 10;
		}
		System.out.println();
	}
}

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L2/04/method.kt
package IR

import java.util.*

internal object method {
    //prog utama
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")
        val input = Scanner(System.`in`)
        val n = input.nextInt()
        //pamggil method
        beautyReverse(n)
    }

    //method reverse
    fun beautyReverse(num: Int) {
        var num = num
        while (num != 0) {
            val r = num % 10
            print(r)
            num = num / 10
        }
        println()
    }
}